WireAPI/Sources/WireAPI/APIs/AccountsAPI/Responses/UpgradeToTeamResponseEnvelopeV7.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+struct UpgradeToTeamResponseEnvelopeV7: Decodable, ToAPIModelConvertible, Sendable {
+
+    /// The team's ID.
+    public let teamId: UUID
+
+    /// The team's name.
+    public let teamName: String
+
+    func toAPIModel() -> UpgradeAccountEnvelope {
+        UpgradeAccountEnvelope(teamId: teamId, teamName: teamName)
```

Comment:
what about?
```suggestion
        UpgradeAccountResponse(teamId: teamId, teamName: teamName)
```
or is Enveloppe a term we use for other endpoints responses?

Reply:
It's used in the endpoint I used for reference, but it seems to be the only one. It's not the response though; it's already something mapped from the actual response.

I'll rename it to `UpgradedAccountTeam` and get rid of the "Enveloppe" suffix.